config

```diff
@@ -1,4 +1,4 @@
 baseuri     https://nodejs.org/dist
 default_variant stretch
-alpine_version  3.11
```

Comment:
I would recommend leaving this like this for a few versions to smooth over the transition of the tags

Reply:
yes, changing this is a breaking change

Reply:
I've just set it back to 3.11